clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/TitusCloudProvider.groovy

```diff
@@ -17,13 +17,14 @@
 package com.netflix.spinnaker.clouddriver.titus
 
 import com.netflix.spinnaker.clouddriver.core.CloudProvider
+import com.netflix.spinnaker.clouddriver.titus.caching.Keys
 import org.springframework.stereotype.Component
 
 import java.lang.annotation.Annotation
 
 @Component
-class TitanCloudProvider implements CloudProvider {
-  final String id = "titan"
-  final String displayName = "Titan"
-  final Class<? extends Annotation> operationAnnotationType = TitanOperation
+class TitusCloudProvider implements CloudProvider {
+  final String id = Keys.PROVIDER
```

Comment:
Keys.PROVIDER is the last vestige of 'titan'. I need to propagate the rename up through the other services before that can become 'titus'